docs/guides/modules.md

```diff
@@ -57,6 +57,54 @@ export default {};
 </script>
 ```
 
+#### Accessing the API from within your extension
+The Directus App's Vue app instance provides a field called `system`, which can be injected into Vue components 
+using [Vue's inject framework](https://v3.vuejs.org/guide/component-provide-inject.html). This `system` field contains functions to access Vuex stores, and more importantly, contains a property called `api`, which is an authenticated Axios instance. Here's an example of how to use it: 
+
+```vue
+<template>
+	<private-view title="Example Collection List">
+		<v-list>
+      <v-list-item v-for="col in collections" v-bind:key="col.collection">{{col.collection}}</v-list-item>
+		</v-list>
+    <v-button v-on:click="logToConsole">Log collections to console</v-button>
+    </private-view>
+</template>
+
+<script>
+export default {
+	data () {
+		return {
+			collections: null
+		}
+	},
+  methods: {
+    logToConsole: function () {
+      console.log(this.collections)
+    }
+  },
+  inject: ['system'],
+  mounted() {
+    // log the system field so you can see what attributes are available under it
+    // remove this line when you're done.
+    console.log(this.system)
+
+    // Get a list of all available collections to use with this module
+    this.system.api.get('/collections?limit=-1')
+      .then((res) => {this.collections = res.data.data});
+  }
+}
+</script>
+```
```

Comment:
```suggestion
```vue
<template>
  <private-view title="Example Collection List">
    <v-list>
      <v-list-item v-for="col in collections" v-bind:key="col.collection">
        {{col.collection}}
      </v-list-item>
    </v-list>
    <v-button v-on:click="logToConsole">Log collections to console</v-button>
  </private-view>
</template>

<script>
  export default {
    data() {
      return {
        collections: null,
      };
    },
    methods: {
      logToConsole: function () {
        console.log(this.collections);
      },
    },
    inject: ["system"],
    mounted() {
      // log the system field so you can see what attributes are available under it
      // remove this line when you're done.
      console.log(this.system);

      // Get a list of all available collections to use with this module
      this.system.api.get("/collections?limit=-1").then((res) => {
        this.collections = res.data.data;
      });
    },
  };
</script>
```
```